.github/workflows/integration.yml

```
@@ -0,0 +1,18 @@
+on:
+  branches:
+    - main
+
+jobs:
+  integration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Setup java
+        uses: actions/setup-java@v1
```

actionlint (1.7.4)

the runner of "actions/setup-java@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)